### ISSUE TITLE
Landing page issues

### ISSUE THREAD

**Issue by artberger** [colinwilson/lotusdocs#128]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for support

### Describe the bug

I was trying out the `landing.yaml` to use it as the landing page for our doc set, and ran into some issues.

- [x] Can’t have book icon in upper left to logo and link to main website (e.g., I’m on [docs.solo.io](http://docs.solo.io/) and want this to go back to [solo.io](http://solo.io/))
- [x] Can’t have the CTA buttons as anchors to content further down the page.
- [x] Can’t add anchors to sections further down the page. At least I tried adding yaml fields for anchors and it didn't seem to do anything.
- [x] Can’t have more than one feature section for different collections of docs. It errors or only displays one of the feature sections.
- [x] In the feature section, can’t click or add a button to go to a page with more info.

Related but different issues:
- https://github.com/colinwilson/lotusdocs/issues/116

### Expected behavior

See the previous list, which I expected based on common landing page patterns.

### Environment

```markdown
- OS: MacOS
- Browser: brave, chrome, safari
- Browser Version: Brave Version 1.58.137, Chromium: 117.0.5938.153, Safari 16.6
```

**Comment by colinwilson** [comment]
@artberger Thanks for this. 👍

I plan to have all the tasks you listed addressed in an update scheduled for Sunday coming.

> Can’t have **more than one feature section for different collections of docs**. It errors or only displays one of the feature sections.

Can you expand on this task a little bit? I'm not sure I understand 100%.

**Comment by artberger** [comment]
@colinwilson thank you for the update!

For the features, we have a couple sets of docs and wanted to have both of them. So right now, I can get something like this for a Gloo Gateway collection:

<img width="1427" alt="image" src="https://github.com/colinwilson/lotusdocs/assets/20074219/8e05b870-9f36-47af-8e2d-76abfc36c317">

But I thought adding another one for Gloo Mesh, such as:

<img width="1430" alt="image" src="https://github.com/colinwilson/lotusdocs/assets/20074219/72926031-1b9c-4970-bb94-a7e620ac4d70">

It doesn't seem I can get them both to show up at the same time; i. e., you can only have one `features` list on the landing page.

**Comment by colinwilson** [comment]
Thanks for clarifying. 👍

Yes! This is exactly what I'm working on rectifying. It will involve a small breaking change to how the sections are defined in the `landing.yaml` data file, but will allow any landing 'section' template to be used multiple times.

**Comment by colinwilson** [comment]
Fixed by #132, #133, #134, #136
